client/Screens/LoginScreen.js

```diff
@@ -26,7 +28,17 @@ export default class LoginScreen extends Component {
       })
     })
     .then((res) => {
-
+      console.log(res);
+      if (res.status === 200) {
```

Comment:
if res.status code === 200 (OK) then we will redirect into the index/root/home screen of the app